Sources/MapLibreSwiftUI/Examples/Camera.swift

```diff
@@ -4,26 +4,27 @@ import SwiftUI
 private let switzerland = CLLocationCoordinate2D(latitude: 46.801111, longitude: 8.226667)
 
 struct CameraDirectManipulationPreview: View {
-    @State private var camera = MapView.Camera.centerAndZoom(switzerland, 4)
+    @State private var camera = MapViewCamera.center(switzerland, zoom: 4)
 
     let styleURL: URL
 
     var body: some View {
         MapView(styleURL: styleURL, camera: $camera)
             .overlay(alignment: .bottom, content: {
-                switch camera {
-                case .centerAndZoom(let coord, let zoom):
-                    Text("\(coord.latitude), \(coord.longitude) z \(zoom ?? 0)")
-                        .padding()
-                        .background(in: RoundedRectangle(cornerRadius: 8), 
-                                    fillStyle: .init())
-                        .padding(.bottom, 42)
-                }
+                Text("\(camera.coordinate.latitude), \(camera.coordinate.longitude) z \(camera.zoom ?? 0)")
+                    .padding()
+                    .background(
+                        Rectangle()
+                            .foregroundColor(.black)
+                            .cornerRadius(8)
+                    )
+
```

Comment:
Without this, it's unreadable black on black ;)

```suggestion
                    .foregroundColor(.white)
```